File: PPE/exercício_02.c
#include <stdio.h>
char str[] = "agiu desproporcionadamente";
int cont = 0;
int tam = 26;
int main(void){
    for(int i = 0; i < tam; i++){
        if(str[i] == 'a')
            cont++;
    }
    printf("A letra 'a' aparece %d vezes na string %s\n", cont, str);
    return 0;
}

File: PPE/exercício_02.py
str = "agiu desproporcionadamente"
cont = 0
tam = 26
def main():
    global cont
    for i in range(tam):
        if (str[i] == 'a'):
            cont += 1
    
    print(f"A letra 'a' aparece {cont} vezes na string {str}")

if __name__ == "__main__":
    main()





